src/shared/codelens/pythonCodeLensProvider.ts

```diff
@@ -272,22 +269,27 @@ export async function initialize({
         })
 
         const debugConfig: PythonDebugConfiguration = makeDebugConfig({ debugPort, samProjectCodeRoot })
-        await invokeLambdaFunction({
-            baseBuildDir,
-            channelLogger,
-            configuration,
-            debugConfig,
-            debugPort: debugConfig.port,
-            samTaskInvoker: taskInvoker,
-            originalSamTemplatePath: args.samTemplate.fsPath,
-            samTemplatePath,
-            documentUri: args.document.uri,
-            originalHandlerName: args.handlerName,
-            handlerName,
-            isDebug: args.isDebug,
-            runtime: args.runtime,
-            telemetryService
-        })
+        await invokeLambdaFunction(
+            {
+                baseBuildDir,
+                originalSamTemplatePath: args.samTemplate.fsPath,
+                samTemplatePath,
+                documentUri: args.document.uri,
+                originalHandlerName: args.handlerName,
+                handlerName,
+                runtime: args.runtime,
+                debugArgs: {
```

Comment:
The change appears to remove the explicit `isDebug` argument, leaving the presence of `debugArgs` to represent whether the action is a run or a debug. `debugArgs` is always passed in currently.

Reply:
✔️